behat-tests/features/acquia_lift_element_variations.feature

```diff
@@ -1,170 +1,198 @@
-#Assumes Marketer role.
+@api @javascript @insulated @campaign
 Feature: Personalize elements variations can be edited for an existing campaign.
   In order to manage element variations in context
   As a site marketer
   I want the ability to add, edit, and delete existing personalize element variations in context.
 
-  @api @javascript @campaign
   Scenario: Add personalize elements to a campaign
+    # I have a campaign.
+    # I login with the marketer role.
+    # I am on an article page.
     Given "acquia_lift" agents:
-      | machine_name                    | label |
+      | machine_name                    | label                           |
       | testing-campaign-add-variations | Testing campaign add variations |
-    And I am logged in as a user with the "Marketer" role
+    And I am logged in as a user with the "access administration pages,access toolbar,administer visitor actions,manage personalized content" permission
```

Comment:
Hmmm - thought we were going to add the code to the FeatureContext to actually create a role from a given statement rather than just showing the permissions.  The reason I hadn't used the permission list before was that I wanted the role definition to be maintained in a single place .  Maybe not so necessary for the non-marketer role, but the "marketer role" should be a bit more centrally defined.